### PR TITLE
Itemfinder

### PR DESCRIPTION
**Pull request by PikalaxALT**

### CODE REVIEW

File: src/itemfinder.c [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+#include "field_player_avatar.h"
+#include "field_specials.h"
+#include "fieldmap.h"
+#include "itemfinder.h"
+#include "constants/songs.h"
+
+static void Task_NoResponse_CleanUp(u8 taskId);
+static void Task_ItemfinderResponseSoundsAndAnims(u8 taskId);
+static void Task_ItemfinderUnderfootSoundsAndAnims(u8 taskId);
+static bool8 HiddenItemIsWithinRangeOfPlayer(struct MapEvents *events, u8 taskId);
```
Comment (jiangzhengwenjz) [inline src/itemfinder.c]: Again inconsistent pointer space style through this file

File: src/itemfinder.c [hunk 2]
```diff
@@ -0,0 +1,659 @@
...
+    const struct MapHeader *mapHeader;
+    u16 localX, localY;
+    u32 localOffset;
+    s32 localLength;
+
+    mapHeader = mapconnection_get_mapheader(connection);
+
+    switch (connection->direction)
+    {
+        // same weird temp variable behavior seen in HiddenItemAtPos
```
Comment (jiangzhengwenjz) [inline src/itemfinder.c]: Why extra indent?

Comment (PikalaxALT) [reply src/itemfinder.c]: Fixed

File: src/itemfinder.c (hunk 1 above)
Comment (PikalaxALT) [reply src/itemfinder.c]: Fixed

**Pull request merged**